SQL/MySQL.sublime-syntax

```diff
@@ -0,0 +1,179 @@
+%YAML 1.2
+---
+name: MySQL
+scope: source.sql.mysql
+version: 2
+extends: Packages/SQL/SQL (basic).sublime-syntax
+
+variables:
+  simple_types: |-
+    (?xi:
+      \b
+      (?:bigint|bigserial|bit|bool|boolean|box|bytea|cidr|circle|date|datetime|double\s+precision|enum|inet|integer|
+        line|longtext|lseg|macaddr|money|ntext|oid|path|point|polygon|real|serial|smallint|sysdate|sysname|text|tinytext|
+        unsigned)
+      \b
+    )
+  types_with_optional_number: |-
+    (?xi:
+      \b
+      (?:bit\s+varying|character\s+(?:varying)?|tinyint|var\schar|float|int|interval|numeric|decimal|times?|timestamp(?:s|tz)?)
+      \b
+    )
+
+contexts:
+  main:
+    - meta_append: true
+    - include: regexps
+
+  comments:
+    - meta_append: true
+    - match: '#'
+      scope: punctuation.definition.comment.sql
+      push: inside-number-sign-comment
+
+  double-dash-comments:
+    - meta_include_prototype: false
+    - match: '--(?=\s)'
+      scope: punctuation.definition.comment.sql
+      push: inside-double-dash-comment
+
+  strings:
+    - meta_append: true
+    - match: '"'
+      scope: punctuation.definition.string.begin.sql
+      push: inside-double-quoted-string
+    - include: begin-interpolation
+
+  inside-double-quoted-string:
+    - meta_include_prototype: false
+    - meta_scope: string.quoted.double.sql
+    - match: '""'
+      scope: constant.character.escape.sql
+    - match: '"'
+      scope: punctuation.definition.string.end.sql
+      pop: 1
+    - include: string-interpolation
+
+  begin-interpolation:
+    - match: '%\{'
+      scope: punctuation.definition.string.begin.sql
```

Comment:
Correct me if I am wrong but does string interpolation in SQL behave the same as normal string interpolations like Python, C# etc. ? If so, then shouldn't this be `punctuation.definition.interpolation.begin.sql` (If so perhaps have a separate scope for leading `%` as `punctuation.definition.interpolation.sql` ?)

Reply:
I have no idea about MySQL interpolations, so I have left them as they are before this PR. I'm hoping someone with more knowledge can improve upon it after this PR is merged.

Reply:
To be honest, I have no idea how interpolations work in MySql. My plan was just to leave it as it is (i.e. before this PR) in the hope that someone with more experience can improve upon it later.